var/spack/repos/builtin/packages/hdf5-vol-async/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Hdf5VolAsync(CMakePackage):
+    """This package enables asynchronous IO in HDF5."""
+
+    homepage = "https://sdm.lbl.gov/"
+    git      = "https://github.com/hpc-io/vol-async"
+    maintainers = ['hyoklee']
+
+    version('v1.0')
+    depends_on('argobots@main')
+    depends_on('hdf5@1.13.0-rc6+mpi+threadsafe')
+
+    def cmake_args(self):
+        """Populate cmake arguments for HDF5 VOL."""
+        args = [
+            '-DBUILD_SHARED_LIBS:BOOL=ON',
+            '-DBUILD_TESTING:BOOL=ON'
```

Comment:
```suggestion
            self.define('BUILD_SHARED_LIBS', True),
            self.define('BUILD_TESTING:BOOL', True),
```

Also don't you usually want to build testing only when `self.run_tests` or doing `spack develop`?

Reply:
Updated according to both suggestions.